apps/condo/domains/common/components/IFrame/IFrame.tsx

```diff
@@ -87,6 +97,23 @@ export const IFrame: React.FC<IFrameProps> = (props) => {
         setFrameHeight(message.height)
     }, [])
 
+    useEffect(() => {
+        const preFetch = async () => {
+            if (isOnClient) {
+                try {
+                    const result = await fetch(pageUrl, { method: 'HEAD' })
+                    if (result.status !== 200) {
```

Comment:
In the case of `/oidc/auth` url the client receives a 302 code.

Reply:
```suggestion
                    if (result.status >= 400) {
```

Reply:
Good take for 302 👍